src/fparser/tests/test_parser.py

```diff
@@ -221,12 +222,23 @@ def test_contains():
 
 
 def test_allocate():
+    ''' Tests for various form of allocate '''
```

Comment:
Sorry, forgot to ask to capitalise "ALLOCATE" here as well!

Reply:
Done.